### ISSUE TITLE
Install Issue

### ISSUE THREAD

**Issue by jpowerz** [DanielnetoDotCom/YouPHPTube#450]
I have an issue that I'm having with the 'mod rewrite'. My hosting server says that they have it enabled but it comes up during the first installation process as it needs to be enabled. I have tried pressing submit after entering the details required to ignore it but it returns back to the installation screen again...
<img width="792" alt="screen shot 2018-03-20 at 11 18 24 am" src="https://user-images.githubusercontent.com/37563595/37631538-4064a070-2c3e-11e8-863e-f7df966ba062.png">

**Comment by birdie60** [comment]
I had the same issue. My solution was a temporarily patch in objects/functions.php:

`function modRewriteEnabled() {
//    return modEnabled("rewrite");
    return true;
}`

After successful installation, I could savely revert to the original file objects/functions.php.

HTH.

**Comment by DanielnetoDotCom** [comment]
Hi, I just remove the mod_rewrite verification, it is not necessary

**Comment by jpowerz** [comment]
I tried that both ways.. First by entering return to true like user 'birdie60' suggested.
I didnt understand too much what you meant Daniel, my apologies as it sounded like you meant that you removed the verification but i tried removing the line and the script didnt work so i just tried the first response and it showed that re write was enabled. 

After it showed that' mod rewrite' was enabled, i filled everything in and clicked submit but it went back into the same install page i started with...

Dont know why it done that.. I created all the mysql user/database info... So i am a little confused now.
